src/nvim/runtime.c

```diff
@@ -1113,6 +1107,18 @@ static int load_pack_plugin(bool opt, char *fname)
   return OK;
 }
 
+static bool fname_in_path(const char *fname, const char *path)
```

Comment:
This function name is a bit too generic. I think this is better (similar naming to `expand_path_option()`):
```suggestion
static bool fname_in_path_option(const char *fname, const char *path_option)
```
And there should be doc comments above this function.

Reply:
I find the `_option` suffix confusing. I generally don't mind generic names on static functions as they have a limited scope, although I don't think it is that generic. If anything it could be more generic, e.g. `entry_in_path()`

Reply:
How do you know that `path` is separate by commas from the function signature?

Reply:
I can add that to the docstring or I can rename to `entry_in_comma_path` or `entry_in_comma_list`?

Reply:
The function uses `path_fnamecmp()`, so `path` should be included in the function name.

> generally don't mind generic names on static functions

Why? When reading a piece of code that calls the function, would you rather have to view the body (or docstring) of the function to understand the code?

Also see this commit: 5dcf2804455f45eac8aad7d900bf60464a4b2888

Reply:
In this case you don't need to read the body.

I'll just revert this. It seems you preferred the readability of the code previously, and you don't seem to have any suggestions on better names.